layers/vxlan.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 Robert Clark. All rights reserved.
```

Comment:
Please use the standard Copyright:


    // Copyright 2016 Google, Inc. All rights reserved.
    //
    // Use of this source code is governed by a BSD-style license
    // that can be found in the LICENSE file in the root of the source
    // tree.